db_schema/queries/v1/spotifyImpressionsFunnel.sql

```diff
@@ -0,0 +1,26 @@
+-- Gets the latest 30-day funnel data (impressions -> considerations -> streams)
+-- Returns exactly 3 rows for the most recent period
+-- Each row includes the total impressions for reference
+SELECT
+	account_id,
+	date,
+	step_id,
+	step_count,
+	conversion_percent,
+	(SELECT step_count 
+	 FROM spotifyImpressionsFunnel sif2 
```

Comment:
the total impression is stored in another table `spotify_impressions` where one row represents a month which should be sufficient for the report. we can also sum the daily values in `daily_impressions` which should be the same. the question is what we really need in the report.